test/functional/client_side_encryption/prose.test.js

```diff
@@ -4,6 +4,18 @@ const chai = require('chai');
 const expect = chai.expect;
 chai.use(require('chai-subset'));
 
+const EJSON = require('mongodb-extjson');
```

Comment:
Can we use EJSON from bson here? and in doing so avoid bringing in the deprecated `mongodb-extjson` lib
`const { EJSON } = require('bson')`